pennylane/operation.py

```diff
@@ -2109,17 +2103,7 @@ def __copy__(self):
 
     def __repr__(self):
         """Constructor-call-like representation."""
-
-        s = " @ ".join([repr(o) for o in self.obs])
-
-        if self.return_type is None:
-            return s
-
-        return (
-            f"{repr(self.return_type)}(wires={self.wires.tolist()})"
-            if self.return_type is qml.measurements.Probability
-            else f"{repr(self.return_type)}({s})"
-        )
+        return " @ ".join([repr(o) for o in self.obs])
```

Comment:
What is the difference here?

Reply:
we include the return type on master (the thing being drawn is an `Observable`, not a `MeasurementProcess`):
```pycon
>>> qml.tape.QuantumScript([], [qml.expval(qml.PauliZ(0) @ qml.PauliZ(1))]).observables[0]
expval(PauliZ(wires=[0]) @ PauliZ(wires=[1]))
```

on this branch:
```pycon
>>> qml.tape.QuantumScript([], [qml.expval(qml.PauliZ(0) @ qml.PauliZ(1))]).observables[0]
PauliZ(wires=[0]) @ PauliZ(wires=[1])
```

this fixes the headache we mentioned while discussing this issue

Reply:
Thanks, got it! This looks great.